modules/sdk/gradle-plugins-workspace/src/gradleTest/formatSourceJakarta/build.gradle

```diff
@@ -0,0 +1,18 @@
+task runGradleTest
+
+gradle.liferayWorkspace {
+	homeDir = new File(buildDir, "bundles")
+}
+
+runGradleTest {
+	dependsOn formatSourceJakarta
+
+	doLast {
+		File file = new File(buildDir.getParentFile(), "modules/foo-web/src/main/java/com/liferay/foo/web/internal/portlet/FooPortlet.java")
```

Comment:
What is the utility of using the `buildDir` here? Why not just reference `project` instead?